app/Http/Controllers/PollController.php

```diff
@@ -30,12 +34,26 @@ public function store(Request $request)
     {
         $poll = Poll::create([
             'name' => $request->name,
-            'url' => $request->url,
-          ]);
-        attach($poll->users(), $request->users);
+            'url' => ""
+        ]);
+        attach($poll->users(), $request->users, "admin");
```

Comment:
It should not be possible to create a poll without users.
Creator of the poll should be marked as owner.

Reply:
Resources page on the wiki was updated yesterday, the admin field is sent for each member of the poll.

I agree that it should be made automatically in a POST request, but we didn't think about this at first.

Anyway we should probably keep it that way so that it will be easier if we ever are to improve this part of the project in the future.

Reply:
Yes we can keep it that way, but the user who creates the poll *must* be a member and admin of the poll it just created. We can't trust user input for that so either we add it automatically in the api or check that it is present in the array.

Reply:
I agree. But if this proves to be too much, we can count on the client app to make correct requests (for now!)

Reply:
No, it will be easy to fix